### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#165] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-30 170823](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/559f0b64-4edd-4ca0-8167-154e007d30d5)

**Comment by daromacarlo** [comment]
![Immagine 2023-11-02 134458](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/448caa34-a521-43a3-9b69-ce52102ea3f1)
![Immagine 2023-11-02 134426](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/f10028e3-0dc8-4403-ab0b-21fe59c373ff)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione analoga a quella in classe
Numerando le monete $1,2,3$ sappiamo che $\mathbb{P}(T_1)=\mathbb{P}(C_1)=1/2$ ; $\mathbb{P}(C_2)=1$ ; $\mathbb{P}(T_3)=1$, inoltre ogni moneta ha la stessa probabilità di essere estratta $1/3$

- **(1)** Lo spazio degli eventi elementari è $\Omega_1=\\{1,2,3\\}$:
$\mathbb{P}_1(T)=\mathbb{P}(1)\mathbb{P}(T_1)+\mathbb{P}(2)\mathbb{P}(T_2)+\mathbb{P}(3)\mathbb{P}(T_3)=1/3\cdot (1/2+0+1)=1/6+1/3=1/2$

- **(2)** Se abbiamo ottenuto testa vuol dire che abbiamo estratto la moneta $1$ o $3$ e l'unico modo per cui ci sia Croce sull'altra faccia è che sia la $1$:
$$\mathbb{P}(1|T)=\mathbb{P}(1|\\{T_1,T_3\\})=\frac{\mathbb{P}(1\cap T_1)+\mathbb{P}(1\cap T_3)}{\mathbb{P}(\\{T_1,T_3\\})}=\frac{\mathbb{P}(1\cap T_1)}{\mathbb{P}(T_1)+\mathbb{P}(T_3)}=\frac{1/6}{1/3\cdot 1/2+1/3\cdot 1}=1/3$$

- **(3)** Estraendo una moneta ed effettuando 2 lanci con essa lo spazio degli eventi dei possibili risultati è $\Omega=\\{(TT),(CC),(TC),(CT)\\}$ (prob non uniforme).
La probabilità di ottenere un'altra Testa sapendo che è uscita al lancio precedente, è la probabilità che esca Testa condizionata dal lanciare le monete $1$ o $3$ che hanno già dato Testa:
$$\mathbb{P}((TT)|T)=\frac{\mathbb{P}((TT)\cap T)}{\mathbb{P}(T)}=\frac{\mathbb{P}(TT)}{\mathbb{P}(T)}=\frac{1/3\cdot 1/4+1/3}{1/2}=5/6$$

**Reply by cornflexxx** [reply]
$3)$

Nel terzo punto credo ci sia un 'errore, infatti intuitivamente  se abbiamo fatto il primo lancio e ottenuto una testa, ci restringiamo al caso in cui abbiamo solo le due monete con doppia testa e testa/croce, e quindi la probabilità di ottenere testa è $3/4$

$$P(T) = P(T|TT)P(TT)+P(T|TC)P(TC)=1\cdot\frac{1}{2}+\frac{1}{2}\cdot \frac{1}{2} = 1/2+1/4 = 3/4$$

**Reply by UnoCasuale** [reply]
Credo che il tuo ragionamento sia giusto ad eccezione, che come calcolato nel punto precedente, la probilità che la moneta sia TT e TC non sia 0.5 bensì 2/3 e 1/3 in quanto è più probabile che la moneta che ci sia rimasta è quella con doppia testa. Quindi credo che la soluzione corretta rimanga 5/6

**Reply by cornflexxx** [reply]
In realtà con $TT$ e $TC$ io intendo le monete proprio , ovvero quella con due teste e quella con una testa e una croce, quindi $P(TT)$ è la probabilità di prendere la moneta doppia testa, quindi essendo solo due  è $1/2$

**Comment by LeoCosta02** [comment]
![IMG_9286](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/f9c51a6a-6234-420a-9e16-431f01549c81)

**Comment by LeoCosta02** [comment]
Il punto due l'ho risolto con la probabilità totale mentre il punto 3 ho visto la soluzione di @CarloDaRomadev e il ragionamento sembrerebbe corretto, il punto 1 mi è venuto uguali ad entrambi

**Comment by daromacarlo** [comment]
Risposta punto 2 + chiarimenti con problema simile presente sul libro consigliato dal professore.

![Immagine 2023-11-07 182821](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/e172075b-4e03-4b1b-bbfb-9e473227f810)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/bc58ac70-98d2-4f46-939c-930b2a4ccfa4)
